### PR TITLE
Added multiple filters support

### PR DESCRIPTION
**Pull request by przegwar**

### CODE REVIEW

File: G1ANT.Addon.Images/Commands/ImageSmoothingFilterCommand.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        { }
+
+        public void Execute(Arguments arguments)
+        {
+            var savingPath = arguments.OutputPath?.Value ?? arguments.Path.Value;
+            var filter = Enum.TryParse(arguments.Filter.Value, out SmoothingFilter filterName) ? smoothingFilterFactory.GetSmoothingFilter(filterName) : throw new ArgumentException("Incorrect filter name");
+
+            using (var image = arguments.Path.OpenImage())
+            {
+                using (var preprocessedImage = PreprocessImageBeforeFiltering(image))
```
Comment (wojszub) [inline G1ANT.Addon.Images/Commands/ImageSmoothingFilterCommand.cs]: you can use many using() one after the other without separate braces

File: G1ANT.Addon.Images/Commands/ImageSmoothingFilterCommand.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        private Bitmap PreprocessImageBeforeFiltering(Bitmap image)
+        {
+            if (image.PixelFormat == PixelFormat.Format24bppRgb)
+                return image;
+
+            if (image.PixelFormat == PixelFormat.Format16bppGrayScale || Image.GetPixelFormatSize(image.PixelFormat) > 32)
+                throw new NotSupportedException("Unsupported image format");
+
+            return AForge.Imaging.Image.Clone(image, PixelFormat.Format24bppRgb);
+        }
```
Comment (wojszub) [inline G1ANT.Addon.Images/Commands/ImageSmoothingFilterCommand.cs]: move to ImageExtensions class and make extension method from this

File: G1ANT.Addon.Images/Factories/SmoothingFilterFactory.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿using AForge.Imaging.Filters;
+using G1ANT.Addon.Images.Enums;
+using System;
+
+namespace G1ANT.Addon.Images.Factories
+{
+    public class SmoothingFilterFactory
+    {
+        public BaseUsingCopyPartialFilter GetSmoothingFilter (SmoothingFilter filter)
```
Comment (wojszub) [inline G1ANT.Addon.Images/Factories/SmoothingFilterFactory.cs]: remove space between method name and bracket

File: G1ANT.Addon.Images/Commands/ImageSmoothingFilterCommand.cs (hunk 2 above)
Comment (przegwar) [reply G1ANT.Addon.Images/Commands/ImageSmoothingFilterCommand.cs]: Removed, because open method sets appropriate image format

**Pull request merged**